test/docs/req.flg

```diff
@@ -0,0 +1 @@
+echo_file test/TEST.properties
```

Comment:
What is this for? I see that there is a similar file for the `jdk` test suite, but it says `echo_file test/TEST.ROOT`.

Reply:
I will admit that I only added it because I saw it in the `jdk` test suit, I wanted to keep the same structure.

Should I remove it?